R/utils-render.R

```diff
@@ -110,6 +110,10 @@ startTemplatedRender <- function(reportJson, author){
   #call custom render function for the report, it should turn a list of HTML fragments that will available to the templates
   templateData[["renderedFragments"]] <- renderCustomFragments(reportJson)
   
+  #adding appropriately formatted start and end dates to reportJSON
+  templateData[['reportMetadata']][['displayStartDate']] <- format(flexibleTimeParse(templateData[['reportMetadata']][['startDate']], templateData[['reportMetadata']][['timezone']]), "%Y-%m-%d")
```

Comment:
From my understanding at this point everything in templateData is used strictly for rendering directly into the HTML so it should be okay to directly modify the startDate and endDate fields. But I'm not absolutely sure that is the case ( @thongsav-usgs ?) and I'm also not against putting it into a new field in order to keep the originally formatted data in case it does need to be rendered into another location or something.